(dp1
S'output'
p2
S"<type 'exceptions.TypeError'> int() argument must be a string or a number, not 'Rows'"
p3
sS'layer'
p4
S'/home/adrian/web2py/applications/CygnusCloud/controllers/main.py'
p5
sS'code'
p6
S"# coding: utf8\n# M\xc3\xa9todo encargado de la p\xc3\xa1gina de inicio de sesi\xc3\xb3n\ndef login(): \n    createAdressBar()\n    auth.settings.login_onaccept = loginAccess\n    #auth.settings.login_next = URL(c='main',f='login')  \n    return dict(form=auth.login())\n    \ndef createAdressBar():\n    response.menu=[[T('Iniciar sesi\xc3\xb3n'),False,URL('login')],\n                   [T('Acerca de'),False,URL('login')]]\n\n@auth.requires_login()                                      \ndef loginAccess(state):\n    rows = userDB(userDB.auth_membership.user_id==auth.user_id).select(userDB.auth_membership.group_id)\n    print rows\n    r = userDB(userDB.auth_group.id == rows).select(userDB.auth_group.role)\n    print r\n    if(r == 'Student'):\n        redirect(URL(c='student',f='runVM'))\n    else:\n        redirect(URL(c='administrator',f='runVM'))\n\nresponse._vars=response._caller(login)\n"
p7
sS'snapshot'
p8
(dp9
sS'traceback'
p10
S'Traceback (most recent call last):\n  File "/home/adrian/web2py/gluon/restricted.py", line 212, in restricted\n    exec ccode in environment\n  File "/home/adrian/web2py/applications/CygnusCloud/controllers/main.py", line 24, in <module>\n  File "/home/adrian/web2py/gluon/globals.py", line 193, in <lambda>\n    self._caller = lambda f: f()\n  File "/home/adrian/web2py/applications/CygnusCloud/controllers/main.py", line 7, in login\n    return dict(form=auth.login())\n  File "/home/adrian/web2py/gluon/tools.py", line 2115, in login\n    callback(onaccept, form)\n  File "/home/adrian/web2py/gluon/tools.py", line 76, in callback\n    [action(form) for action in actions]\n  File "/home/adrian/web2py/gluon/tools.py", line 2929, in f\n    return action(*a, **b)\n  File "/home/adrian/web2py/applications/CygnusCloud/controllers/main.py", line 17, in loginAccess\n    r = userDB(userDB.auth_group.id == rows).select(userDB.auth_group.role)\n  File "/home/adrian/web2py/gluon/dal.py", line 8905, in select\n    return adapter.select(self.query,fields,attributes)\n  File "/home/adrian/web2py/gluon/dal.py", line 1618, in select\n    sql = self._select(query, fields, attributes)\n  File "/home/adrian/web2py/gluon/dal.py", line 1478, in _select\n    sql_w = \' WHERE \' + self.expand(query)\n  File "/home/adrian/web2py/gluon/dal.py", line 1311, in expand\n    return op(first, second)\n  File "/home/adrian/web2py/gluon/dal.py", line 1239, in EQ\n    self.expand(second, first.type))\n  File "/home/adrian/web2py/gluon/dal.py", line 1321, in expand\n    return str(self.represent(expression,field_type))\n  File "/home/adrian/web2py/gluon/dal.py", line 1746, in represent\n    return str(int(obj))\nTypeError: int() argument must be a string or a number, not \'Rows\'\n'
p11
s.